src/main/scala/diplomaticobjectmodel/DiplomaticObjectModelUtils.scala

```diff
@@ -225,9 +225,21 @@ object DiplomaticObjectModelAddressing {
       grandParentOpt = b.device.get.parent
       gp <- grandParentOpt
     } yield OMInterrupt(
-            receiver = getDeviceName(gp),
-            numberAtReceiver = getInterruptNumber(b.value),
-            name = name
-          )
-    }
-}
+      receiver = getDeviceName(gp, resources),
+      numberAtReceiver = getInterruptNumber(b.value),
+      name = name
+    )
+  }
+
+  def describeGlobalInterrupts(name: String, resources: ResourceBindings): Seq[OMInterrupt] = {
+    val bindings = resources("int")
```

Comment:
This method name includes "global", but I'm not sure I see where you separate global interrupts from local ones.